quickstarts/kit/docker-image/pom.xml

```diff
@@ -20,7 +20,7 @@
 
   <groupId>org.eclipse.jkube.quickstarts.kit</groupId>
   <artifactId>docker-image</artifactId>
-  <version>1.5.1</version>
+  <version>1.6.0-SNAPSHOT</version>
```

Comment:
What is the rules for quickstarts here?
I guess we switch to -SNAPSHOT if the test is impacted and move to the released version at release time.

Reply:
Yep.

Quickstarts should always refer to the latest **released** version.
However, if the current version becomes incompatible with the released version (!breaking changes), then it must refer to the next SNAPSHOT.

During the release process, there's a step to set the quickstarts version to the currently released version.